### PR TITLE
Provided and remaining budget on milestones.list and milestones.info [PRO-1935]

### PR DESCRIPTION
**Pull request by adognameddog**

### CODE REVIEW

File: src/changes-backwards-compatible.apib [hunk 1]
```diff
@@ -10,6 +10,11 @@ We list all backwards-compatible additions here. These are currently available i
 - We added an optional filter on `ids` to `invoices.list`.
 - We added `creditNote.booked` and `product.added` to the available webhooks list.
 
+#### June 2019
```
Comment (stivni) [inline src/changes-backwards-compatible.apib:15]: I'm not sure how this fits the order of things here :)

**Review by jenssegers — DISMISSED** [review]

File: src/07-projects/milestones.apib [hunk 2]
```diff
@@ -89,6 +93,10 @@ Get details for a single milestone.
                     + time_and_materials
                     + fixed_price
                     + non_invoiceable
+            + `budget` (object) - This is only available for users that have access to budgets on projects.
+                + `allocated` (Money)
```
Comment (stivni) [inline src/07-projects/milestones.apib]: as discussed in person, I now think that "allocated" actually creates confusion around "is this allocated *to* the project, or allocated *within* the project"...

_provided_ budget might seem to be a good alternative...

Comment (jenssegers) [reply src/07-projects/milestones.apib]: Isn't "allocated" the estimated budget?

Comment (jenssegers) [reply src/07-projects/milestones.apib]: Also, if it's between allocated and provided I would choose allocated.

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by ketlane — APPROVED** [review]

File: src/changes-backwards-compatible.apib (hunk 1 above)
Comment (ketlane) [reply src/changes-backwards-compatible.apib:15]: I guess we can reevaluate before merging

File: src/07-projects/milestones.apib (hunk 2 above)
Comment (adognameddog) [reply src/07-projects/milestones.apib]: After some research we think "provided budget" for a milestone (so when you create a milestone you specify that budget) is better than "allocated". Allocated in this case means the budget you plan to spend, so haven't spent yet.

**Review by joaoinacio — APPROVED** [review]

**Review by jenssegers — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Review by stivni — APPROVED** [review]

**Pull request closed without merging**